src/uct/ib/dc/dc_mlx5.c

```diff
@@ -205,8 +205,6 @@ static ucs_status_t uct_dc_mlx5_iface_query(uct_iface_h tl_iface, uct_iface_attr
         iface_attr->cap.flags &= ~(UCT_IFACE_FLAG_ERRHANDLE_PEER_FAILURE |
                                    UCT_IFACE_FLAG_ERRHANDLE_ZCOPY_BUF    |
                                    UCT_IFACE_FLAG_ERRHANDLE_REMOTE_MEM);
-    } else {
-        iface_attr->cap.flags |= UCT_IFACE_FLAG_EP_CHECK;
```

Comment:
do we really need/want to remove it?
if yes should be another PR to remove DC ep_check altogether including the implementation

Reply:
leftover - returned back, we shouldn't remove it, since it is needed for `UCX_TLS=dc_mlx5` users